model/model_test.go

```diff
@@ -47,3 +47,57 @@ func TestAccount_Balance(t *testing.T) {
 	require.Equal(t, Balance{}, account.Balance("A"))
 	require.Equal(t, Balance{Tick: "B", Free: 1.1, Lock: 1.3}, account.Balance("B"))
 }
+
+func TestHeikinAshi_CalculateHeikinAshi(t *testing.T) {
+	ha := NewHeikinAshi()
+
+	if (HeikinAshi{}.PreviousHACandle != Candle{}) {
+		t.Errorf("PreviousCandle should be empty")
+	}
+
+	// BTC-USDT weekly candles from Binance from 2017-08-14 to 2017-10-09
+	// First market candles were used to easily test accuracy against TradingView without having to download all market data.
```

Comment:
The linter (lll) requires a break line here, can you split this comment in multiple lines?